server/webapp/WEB-INF/rails/app/presenters/api_v1/admin/merged_environments/pipeline_config_summary_representer.rb

```diff
@@ -26,15 +26,15 @@ def initialize(options)
         end
 
         link :self do |opts|
-          opts[:url_builder].apiv5_admin_pipeline_url(@pipeline.getName())
+          "#{opts[:url_builder].default_url_options[:protocol]}://#{opts[:url_builder].default_url_options[:host]}/api/admin/pipelines/#{@pipeline.getName()}"
```

Comment:
`default_url_options` are just that — defaults used when the incoming request does not contain the required data. As such `default_url_options` can be blank. See `ApiV1::Dashboard::PipelineRepresenter` for how to do this right.